modules/using-mediated-devices.adoc

```diff
@@ -0,0 +1,9 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/advanced_vm_management/virt-configuring-mediated-devices.adoc
+
+:_content-type: CONCEPT
+[id="virt-using-mediated-devices"]
+= Using mediated devices
+
+Using mediated devices.
```

Comment:
Can this be fleshed out a little bit to a more complete intro sentence?

Reply:
@jeana-redhat I had the same idea when I looked at this. Let me re-look at that.